server/src/main.rs

```diff
@@ -0,0 +1,17 @@
+mod daemon;
```

Comment:
I don't understand the reason for having this in a separate mod; it is not like we are going to expose it as a separate library anyways. So drop the folder. initially, i was suggesting to name "server" as "daemon" instead of adding an extra mod.

Reply:
sorry for the mix up. this is fixed now.